projects/plugins/jetpack/extensions/blocks/tiled-gallery/editor.scss

```diff
@@ -201,9 +201,3 @@
 		box-shadow: 0 0 0 $active-item-outline-width $gray-600 !important;
 	}
 }
-
-
-.block-editor-block-inspector .jetpack-tiled-gallery-url-notice {
-	color: #ff0000;
-	padding-bottom: $grid-unit-05;
-}
```

Comment:
I believe we are missing an extra line here :) Nitpicking

Reply:
Oh that's confusing, my editor is showing the extra line, when I check out this PR - line 204 is a blank line. I think the linting actions would have flagged this if there was actually a missing line here? Or am I wrong on that? 🤔 What do you see when checking this PR out locally?

Reply:
You are right, when checking out the PR locally I see the extra line 👍